incubator/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/KafkaGrpcBinding.java

```diff
@@ -23,6 +26,7 @@
 public final class KafkaGrpcBinding implements Binding
 {
     public static final String NAME = "kafka-grpc";
+    private final Long2ObjectHashMap<AtomicInteger> workers = new Long2ObjectHashMap<>();
```

Comment:
This needs to be a `ConcurrentMap<Long, AtomicInteger>` assigned to `new ConcurrentHashMap<>()`.

Otherwise the `Long2ObjectHashMap` can misbehave across threads during initialization.

Reply:
Yeah, thanks I changed it seems like got changed during the stash